guides/products-and-variants/overview.md

```diff
@@ -0,0 +1,85 @@
+# Overview of products and variants
+
+Products and variants are integral to Solidus. While the `Spree::Product` and
+`Spree::Variant` models depend on each other, it is important to understand how
+they are different:
+
+- `Spree::Product`s track the general information about a product. This includes
```

Comment:
We're talking about products and variants here using the same terms. I think it would be best to start with examples of cases where a product has variants, and what the product is and what the variant is.

Reply:
You are very right. I've extended the products and variants bullet points here.